dist/save/index.js

```diff
@@ -41063,33 +41063,34 @@ const utils = __importStar(__webpack_require__(443));
 process.on("uncaughtException", e => utils.logWarning(e.message));
 function saveImpl(stateProvider) {
     return __awaiter(this, void 0, void 0, function* () {
+        let cacheId;
         try {
             if (!utils.isCacheFeatureAvailable()) {
-                return;
+                return 0;
```

Comment:
0 may be a real cacheId as DB entry starts at 0. Should we use something like -2

Reply:
Yes we can. I was thinking, in the future we can also map the negative numbers to some error types to decide how to log/show error to the user.

Reply:
Used -2 now.

Reply:
Lets do that now itself. We can define an enum on cache return type.

Reply:
Actually re-thinking about this. Why do we need to return id here. Can we return error here. Any error and SaveOnly should print warning message. No?

Reply:
Does this mean we also need to implement error handling in save.ts?